README.md

```diff
@@ -40,7 +40,8 @@ The page will reload if you make edits. You will also see any lint errors in the
 Before submitting a PR/MR make sure your code is compliant with our JS rules by running: `npm run test:js`
 You can format it automatically by running: `npm run format`.
 
-To make sure your SCSS is compliant run: `npm run test:scss`
+We use the SASS linter because it is also compatible with SCSS syntax.To make sure your SCSS is compliant
```

Comment:
can you add a space after the first full stop?

`syntax. To`

Reply:
Added